spec/pluralrules.html

```diff
@@ -19,24 +19,14 @@
           1. Let _options_ be ObjectCreate(*%ObjectPrototype%*).
         1. Else
           1. Let _options_ be ? ToObject(_options_).
-        1. Let _opt_ be a new Record.
-        1. Let _s_ be ? GetOption(_options_, *"type"*, *"string"*, « *"cardinal"*, *"ordinal"* », *"cardinal"*).
-        1. Set _pluralRules_.[[Type]] to _s_.
-        1. Let _mnid_ be ? GetNumberOption(_options_, *"minimumIntegerDigits"*, 1, 21, 1).
-        1. Set _pluralRules_.[[MinimumIntegerDigits]] to _mnid_.
-        1. Let _mnfd_ be ? GetNumberOption(_options_, *"minimumFractionDigits"*, 0, 20, 0).
-        1. Set _pluralRules_.[[MinimumFractionDigits]] to _mnfd_.
-        1. Let _mxfd_ be ? GetNumberOption(_options_, *"maximumFractionDigits"*, _mnfd_, 20, max(_mnfd_, 3)).
-        1. Set _pluralRules_.[[MaximumFractionDigits]] to _mxfd_.
-        1. Let _mnsd_ be ? Get(_options_, *"minimumSignificantDigits"*).
-        1. Let _mxsd_ be ? Get(_options_, *"maximumSignificantDigits"*).
-        1. If _mnsd_ is not *undefined* or _mxsd_ is not *undefined*, then
-          1. Let _mnsd_ be ? GetNumberOption(_options_, *"minimumSignificantDigits"*, 1, 21, 1).
-          1. Let _mxsd_ be ? GetNumberOption(_options_, *"maximumSignificantDigits"*, _mnsd_, 21, 21).
-          1. Set _pluralRules_.[[MinimumSignificantDigits]] to _mnsd_.
-          1. Set _pluralRules_.[[MaximumSignificantDigits]] to _mxsd_.
-        1. Let _localeData_ be *%PluralRules%*.[[LocaleData]].
-        1. Let _r_ be ResolveLocale(*%PluralRules%*.[[AvailableLocales]], _requestedLocales_, _opt_, *%PluralRules%*.[[RelevantExtensionKeys]], _localeData_).
+        1. Let _t_ be ? GetOption(_options_, *"type"*, *"string"*, « *"cardinal"*, *"ordinal"* », *"cardinal"*).
```

Comment:
If I understand correctly, now instead of having `opt`, we store them into internal slots. Is that correct?